package/gluon-state/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-state
```

Comment:
Something more descriptive like `gluon-state-check` would be preferable.

Reply:
On first thought that repeats what it does unnecessarily. Either gluon-state or gluon-check.

Reply:
Like gluon-state-online? Just kidding...
Renamed once again in 8a813a7cf25cf9078209323b8a0b7d663cc56f1c

Reply:
@AiyionPrime 

> Like gluon-state-online? Just kidding...

That makes much more sense to me... There could also be gluon-state-meshlink or something.

Reply:
It's also much more descriptive... gluon-state-check - what state?

Reply:
I'd call it gluon-check-online

Reply:
I'd call it `gluon-state-check`. As it is now.

Reply:
Fine with me. I consider this resolved. Objections?